test/expo/features/support/env.rb

```diff
@@ -0,0 +1,130 @@
+require 'json'
+
+require_relative '../lib/browserstack_driver'
+
+BROWSER_STACK_URI = "https://api-cloud.browserstack.com/app-automate/upload"
+
+@bs_username = ENV['BROWSER_STACK_USERNAME']
+@bs_access_key = ENV['BROWSER_STACK_ACCESS_KEY']
+@bs_local_id = ENV['BROWSER_STACK_LOCAL_IDENTIFIER'] || 'mazzzzeee'
+
+@bs_app_url = nil
+
+$bs_driver = nil
+
+def device_type
+  ENV['DEVICE_TYPE']
+end
+
+def upload_app
+  res = `curl -u "#{@bs_username}:#{@bs_access_key}" -X POST "#{BROWSER_STACK_URI}" -F "file=@#{ENV['APP_LOCATION']}"`
+  resData = JSON.parse(res)
+  if resData.include?('error')
+    puts "BrowserStack upload failed due to error: #{resData['error']}"
+    exit(false)
+  else
+    @bs_app_url = resData['app_url']
+  end
+end
+
+def start_driver
+  unless $bs_driver
+    $bs_driver = BSAppAutomator::Driver.new(device_type, @bs_app_url, @bs_username, @bs_access_key, @bs_local_id)
+    $bs_driver.start_local
+    $bs_driver.start_driver
+  end
+end
+
+def wait_on_element(element)
+  unless $bs_driver.nil?
+    $bs_driver.wait_for_element(element)
+  end
+end
+
+def click_element(element)
+  unless $bs_driver.nil?
+    $bs_driver.click_element(element)
+  end
+end
+
+def set_dropdown_value(element, value)
+  unless $bs_driver.nil?
+    $bs_driver.click_element(element)
+    sleep(1)
```

Comment:
do we really need the sleep?

Reply:
We actually don't need this at all, so I'll remove it.